lib/SlackBot.js

```diff
@@ -78,7 +78,7 @@ function Slackbot(configuration) {
           // this won't work for single team bots because the team info
           // might not be in a db
           if (err || !team) {
-            slack_botkit.log('Received slash command, but could not load team');
+            slack_botkit.logerror('Received slash command, but could not load team');
```

Comment:
I did this. `slack_botkit.log` is still a function, but now it has methods for each severity as well.